### PR TITLE
Refactor to use Hooks

### PR DESCRIPTION
**Pull request by abdallahmalima**
## In this part I:

- Refactor everything to use Hooks and functional components

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,63 +1,87 @@
...
+    this.state = {
+      calculateObject: {
+        total: null,
+        next: null,
+        operation: null,
+
+      },
+    };
+  }
+
```
Comment (KDlamini) [inline src/components/Calculator.js:18]: Hey @abdallahmalima

The objective of this milestone was to refactor the Calculator component from a class-based component to a functional component using [hooks](https://reactjs.org/docs/hooks-intro.html). Your Calculator component is still class-based.

Here's an example of how it should look like:
`// ./src/components/Calculator.js`
```
import React, { useState } from 'react';

const Calculator = () => {
     const [state, setState] = useState(
     {
        total: null,
        next: null,
        operation: null,
      });

     handleClick=(e) => {
      //Update state using setState
     }

     return (
      <div>
       // Calculator app design goes here
      </div>
     )
};

export default Calculator; 
```

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @abdallahmalima 🙋‍♀️,  

Great work so far! 💯 
There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Professional pull request
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (rloterh) [reply src/components/Calculator.js]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Antony,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Correctly refactored Calculator component using _hooks_.
- App works correctly after component refactor.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- Kindly ensure to add the _Future Features_ section to your ReadMe, which should include 1-3 potential features to be added
- It's recommended that you add an MIT license, or any other similar license to your ReadMe for this project.
Implementing the above change(s) will make your ReadMe/ repository stand out even better 😉

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Pull request merged**